Source/EyesRelax/DefaultSettings.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace EyesRelax
+{
+    public class DefaultSettings
+    {
+        protected TimeSpan workingTimeMinutes = new TimeSpan(0, 25, 0);
+        protected int workingTime;
+        public int WorkingTime { get => workingTime = (int)workingTimeMinutes.TotalMinutes; }
```

Comment:
I think we can simplify it. 
First thing, maybe it would be better to have only properties. We don't need fields behind them. (I mean for `WorkingTime` property we don't need `workingTime` field, we can just return in getter `(int)workingTimeMinutes.TotalMinutes`)